environments/dotnet/README.md

```diff
@@ -163,6 +163,60 @@ $ curl "http://$FISSION_ROUTER/add?x=30&y=12"
   42
 ```
 
+## Accessing Http request information example
+
+### Setup fission environment
+First you need to setup the fission according to your cluster setup as 
+specified here: https://github.com/fission/fission
+
+
+### Create the class to run
+
+Secondly you need to create a file /tmp/func.cs containing the following code:
+
+```
+using System;
+using Fission.DotNetCore.Api;
+
+public class FissionFunction
+{
+    public string Execute(FissionContext context){
```

Comment:
I have added sample on how to deserialize a json request body into objects and I have added a convenience method that will give you the request body as a string